source/summary.rst

```diff
@@ -38,40 +51,54 @@ already implemented; we link to the repositories in the appropriate places.
 Attack model and terminology
 ++++++++++++++++++++++++++++
 
-We consider a *network adversary* which can access and modify network messages.
-Examples of this adversary can be an ISP, an e-mail provider, an AS,
+We consider a *network adversary* that can read, modify, and create
+network messages.
+Examples of such an adversary are an ISP, an e-mail provider, an AS,
 or an eavesdropper on a wireless network.
-The goal of the adversary is to perform a machine-in-the-middle attack
-on the key exchange within peers. This would enable her to
-i) read the content of messages, and ii) impersonate peers.
-We presume that all peers are honest. They do not collaborate with the adversary and follow the protocols described in this document.
-
-Peers need access to a *trusted*, *out-of-band* channel of communication
-that is not visible to the adversary, and thus cannot be manipulated.
-This channel can be used
-from end-user key verification or key authentication protocols,
-without fear of intervention by the adversary.
-
-Problems of current key verification techniques
+The goal of the adversary is to i) read the content of messages, ii)
+impersonate peers -- communication partners, and iii) to learn who communicates
+with whom.
+To achieve these goals,
+an active adversary might try, for example,
+to perform a machine-in-the-middle attack on the key exchange protocol
+between peers.
+
+Peers learn the content of the messages.
+Therefore,
+we assume that all peers are honest.
```

Comment:
stumbled on this -- probably better to have a single combined sentence instead of two here.